abs.yaml

```
@@ -0,0 +1,4 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/qtbase-feedstock/pr7/1550c51
```

```suggestion
  - https://staging.continuum.io/prefect/fs/qtbase-feedstock/pr7/01c13ff
```
Newest `qtbase` puts `qt6.conf` in the non-devel package.